source/deployment/permissions-backend.rst

```diff
@@ -0,0 +1,98 @@
+Advanced Permissions: Backend Infrastructure
+=============================================
+
+This document outlines the backend server infrastructure for permissions in Mattermost and is recommended only for technical Admins or developers looking to make modifications to their installation.
+
+
+.. note::
+
+  The contents of this document apply to Mattermost Server version 5.0 and later. 
+
+
+.. contents::
+  :backlinks: top
+  :local:
+  
+Entity Definitions
```

Comment:
While what you write hear makes sense and is well written, I'm a visual learner, so a visual (or visuals) showing the relationship between these entities would be a huge upgrade to this section.

Reply:
e.g. I get the hierarchy between  permissions > roles > scope pretty easily, but then context and schemes took a moment to ready and really understand.

Reply:
Or another option is having an example at the end that runs through each of these. 0/5